lib/flex.js

```diff
@@ -59,18 +59,17 @@ const processWorkflow = async (context, registry, config) => {
   await processPreActions(context, registry, config)
 
   // process each rule found in configuration
-  config.settings.forEach(rule => {
+  for (const rule of config.settings) {
```

Comment:
What's the reasoning for changing this at all?

Reply:
right now, it's just doing `fire and forget`, I don't know how to test that at all since the test would end before the api has been hit. I would love to hear if you have a way to test it

Reply:
I'm reffering to the for loop syntax -- is there a difference in doing`forEach` vs. `for` ?

Reply:
you can't do proper `async/await` in forEach because of the anynomous functions

Reply:
Unless you collect the calls as Promise and do `await` on the Promises